hack/update-import-aliases.sh

```diff
@@ -22,7 +22,7 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${SCRIPT_ROOT}"
 ROOT_PATH=$(pwd)
 
-GO111MODULE=on go install "golang.org/x/tools/cmd/goimports@v0.1.5"
+GO111MODULE=on go install "golang.org/x/tools/cmd/goimports"
```

Comment:
keepping it same with https://github.com/karmada-io/karmada/blob/ffdc58cbba4752b3db2d0a0c3c42cfa45cd159c4/hack/update-estimator-protobuf.sh#L27

/lgtm